circuit-types/src/traits.rs

```diff
@@ -359,6 +359,18 @@ impl BaseType for u64 {
     }
 }
 
+impl BaseType for usize {
```

Comment:
do we still need to `impl BaseType for u64`?

Reply:
yeah a few other fields in witness types make more sense as a u64 (numeric) vs usize (index)